src/parse/asp/interpreter.go

```diff
@@ -513,7 +513,11 @@ func (s *scope) interpretFString(f *FString) pyObject {
 	var b strings.Builder
 	for _, v := range f.Vars {
 		b.WriteString(v.Prefix)
-		b.WriteString(s.Lookup(v.Var).String())
+		if v.Config != "" {
+			b.WriteString(s.config.MustGet(v.Config).String())
```

Comment:
Not very comfortable. I think it should blow up if I misspell the config key in the fstring - same as it would if I wrote `CONFIG.BLAH` normally.

Why're you unhappy with it?

Reply:
I was under the impression that python didn't error if you gave it a name it didn't know about, but it does so I'm actually ok with this

Reply:
Ah nice. Thanks.